client/main/analyses.js

```diff
@@ -121,19 +121,19 @@ Analysis.prototype.setOptions = function(values) {
     }
 };
 
-Analysis.prototype.renameColumns = function(columnRenames) {
+Analysis.prototype.renameColumns = function(columnRenames, silent) {
```

Comment:
this is only called from once place and silent is always true ... do you want to ditch the silent argument?

the name `renameColumns()` implies a "doing" (i.e. that this function will rename columns), where as this is more of a "responding to". how about `notifyColumnsRenamed()`?